docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.theme.colors.scss

```diff
@@ -0,0 +1,15 @@
+// I cannott call these teal, purple, etc because sass tries to parse it beforehand
+// All named colors are reserved values in Sass and will be turned into hex values regardless of placement and use.
+// so I prefixed everything with "su-"
```

Comment:
On second thought, `su-` is used as a decanter prefix, so using it here might create some confusion. Maybe it's better to just do `color-teal`, `color-mint`, etc.